Lib/fontTools/otlLib/builder.py

```diff
@@ -348,6 +348,52 @@ def build(self):
         # buildLookup_
         return self.buildLookup_(subtables)
 
+    def buildFormat1Subtable(self, ruleset, chaining=True):
+        st = self.newSubtable_(chaining=chaining)
+        st.Format = 1
+        st.populateDefaults()
+        coverage = set()
+        rulesets_by_first_glyph = {}
+        ruleAttr = self.ruleAttr_(format=1, chaining=chaining)
+
+        for rule in ruleset.rules:
+            rule_table = self.newRule_(format=1, chaining=chaining)
+
+            if chaining:
+                rule_table.BacktrackGlyphCount = len(rule.prefix)
+                rule_table.LookAheadGlyphCount = len(rule.suffix)
+                rule_table.Backtrack = [list(x)[0] for x in reversed(rule.prefix)]
+                rule_table.LookAhead = [list(x)[0] for x in rule.suffix]
+
+                rule_table.InputGlyphCount = len(rule.glyphs)
+            else:
+                rule_table.GlyphCount = len(rule.glyphs)
+
+            rule_table.Input = [list(x)[0] for x in rule.glyphs[1:]]
+
+            self.buildLookupList(rule, rule_table)
+
+            firstGlyph = list(rule.glyphs[0])[0]
+            if not firstGlyph in rulesets_by_first_glyph:
```

Comment:
"if item **not** in collection" is the correct way, not "if **not** item in collection"